app/views/admin/dashboard/index.html.slim

```diff
@@ -1 +1,6 @@
-h1 Dashboard
+- content_for :title
+  = 'Dashboard'
```

Comment:
To just output plain text, you can start the line with `|`, e.g. `| Dashboard`. With such a small thing, though you could also combine it into the above line: `- content_for :title, Dashboard`